app/presenters/publishing_api_presenters/contact.rb

```diff
@@ -0,0 +1,121 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::Contact
```

Comment:
Could this inherit from Item, like the other presenters do? If the differences are too great, it would be good to explain that somewhere.